src/ErrorHandling/AbortWithErrorMessage.cpp

```diff
@@ -22,7 +23,8 @@ void abort_with_error_message(const char* expression, const char* file,
      << message << "\n"
      << "############ ASSERT FAILED ############\n"
      << "\n";
-  Parallel::abort(os.str());
+  Parallel::printf(os.str());
+  Parallel::abort("");
 }
```

Comment:
Comment that this change enables the error message to print even if the code segfaults, or if I misunderstand, add a comment explaining why this is changed.